driver/syscall_table.c

```diff
@@ -1350,6 +1356,9 @@ const enum ppm_syscall_code g_syscall_ia32_code_routing_table[SYSCALL_TABLE_SIZE
 #ifdef __NR_ia32_semctl
 	[__NR_ia32_semctl - SYSCALL_TABLE_ID0] =  PPM_SC_SEMCTL,
 #endif
+#ifdef __NR_ia32_semget
+	[__NR_ia32_semget - SYSCALL_TABLE_ID0] =  PPM_SC_SEMget,
```

Comment:
thank you @luca3m ! I am shocked that this compiles too.. but it does (definitely a search and replace mistake though).